build.yaml

```diff
@@ -1021,6 +1029,7 @@ libs:
   - grpc++_codegen_base_src
   - grpc++_codegen_proto
   - grpc++_config_proto
+  - thrift_util
```

Comment:
Can you put in some thrift tests as part of this PR? Preferably a really simple one. It should live in tools/grift/tests or something like that, rather than test/cpp . It would be helpful so that we can run tests to make sure that thrift support doesn't get broken.